src/docs/truffle/getting-started/running-migrations.md

```diff
@@ -174,9 +175,9 @@ The deployer contains many functions available to simplify your migrations.
 
 ### deployer.deploy(contract, args..., options)
```

Comment:
This notation threw me off because it looked odd in JavaScript context. Maybe a note to explain our notation? or highlight that truffle will do some magic to infer the arguments?

Reply:
Should be `...args` instead, then it's normal JS syntax

Reply:
It wouldn't be legal JS. (Rest parameters should be the last formal param).

Reply:
they haven't changed that yet??

Reply:
@cds-amal It isn't meant to be a literal JS `...args`. If you are doing a deployment and the constructor takes 3 arguments, your function call will look like 
```
await deployer.deploy(MyContract, firstArg, secondArg, thirdArg, { from: accounts[3] });
```
But if it only takes one argument, then it would be
```
await deployer.deploy(MyContract, firstArg, { from: accounts[3] });
```

Reply:
Thanks for the explanation. I guess the `args...` should be enough of a hint that it's not JavaScript